### ISSUE TITLE
Weird error "AttributeError: 'str' object has no attribute 'query'" when running convert

### ISSUE THREAD

**Issue by NikolayOskolkov** [SPAAM-community/AMDirT#54]
This error seems to be on both master and dev branches:

```
(AMDirT) nikolay@dell:~/WABI/Misc/SPAAM/AMDirT$ AMDirT convert test/valid.tsv ancientmetagenome-hostassociated
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/AMDirT", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/AMDirT/cli.py", line 93, in convert
    run_convert(**kwargs)
  File "/usr/local/lib/python3.6/dist-packages/AMDirT/convert/main.py", line 31, in run_convert
    supported_archives=supported_archives,
  File "/usr/local/lib/python3.6/dist-packages/streamlit/legacy_caching/caching.py", line 573, in wrapped_func
    return get_or_create_cached_value()
  File "/usr/local/lib/python3.6/dist-packages/streamlit/legacy_caching/caching.py", line 557, in get_or_create_cached_value
    return_value = func(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/AMDirT/core/utils.py", line 90, in prepare_eager_table
    samples.query("archive in @supported_archives")
AttributeError: 'str' object has no attribute 'query'

```

**Comment by NikolayOskolkov** [comment]
The error seems to be coming from the main.py / run_convert function in "convert" part of AMDirT. It seems to be missing the data frame samples, it is never read into memory, so there is no line like:

samples = pd.read_csv("test/valid.tsv", sep="\t")

the test/valid.tsv is probably read somewhere in the streamlit part but I can't figure out where and why it does not reach the run_convert function.

**Comment by NikolayOskolkov** [comment]
It looks like it can be solved by adding this to main.py in convert:

`samples = pd.read_csv(samples, sep="\t")`

I also added these lines to main.py in convert:

```
if output is None:
        eager_table.to_csv(f"eager_input_table.tsv", sep="\t", index=False)
        accession_table.to_csv(f"fetchNGS_input_table.tsv", sep="\t", header=False, index=False)
    else:
        eager_table.to_csv(f"{output}/eager_input_table.tsv", sep="\t", index=False)
        accession_table.to_csv(f"{output}/fetchNGS_input_table.tsv", sep="\t", header=False, index=False)
```

this is because the "output" argument of run_convert can be None.

**Comment by maxibor** [comment]
This should have been fixed with #55
